ci/requirements-2.7-0.19.2.pip

```diff
@@ -5,3 +5,4 @@ google-auth-oauthlib
 PyCrypto
 python-gflags
 mock
+google-cloud-bigquery==0.26.0
```

Comment:
I would just add the dep, no need to fix a version in *every* test version. leave some of the empty so that they get the latest, maybe leave the 3.5 as is (as it already pins some other things).

Reply:
Good point, updated.